package.json

```diff
@@ -52,7 +52,9 @@
     "@types/lodash": "^4.14.149",
     "@types/react": "^16.9.19",
     "@types/react-dom": "^16.9.5",
-    "babel-eslint": "^6.1.2",
+    "@typescript-eslint/eslint-plugin": "^2.19.0",
+    "@typescript-eslint/parser": "^2.19.0",
+    "babel-eslint": "^10.0.3",
```

Comment:
Had to upgrade `babel-eslint`, `eslint`, and a few others in order to get the `typescript-eslint` packages fully working.